Documents/CommitteeSpecifications/2.1.0/sarif-schema-2.1.0.json

```diff
@@ -2354,7 +2354,7 @@
           "description": "The language of the messages emitted into the log file during this run (expressed as an ISO 639-1 two-letter lowercase culture code) and an optional region (expressed as an ISO 3166-1 two-letter uppercase subculture code associated with a country or region). The casing is recommended but not required (in order for this data to conform to RFC5646).",
           "type": "string",
           "default": "en-US",
-          "pattern": "^[a-zA-Z]{2}|^[a-zA-Z]{2}-[a-zA-Z]{2}]?$"
+          "pattern": "^[a-zA-Z]{2}|^[a-zA-Z]{2}-[a-zA-Z]{2}?$"
```

Comment:
>pattern [](http://example.com/codeflow?start=11&length=7)

Ouch, yes. I think we need to fix this in the errata and then update schemastore.org. We'll need to ask @dmk42 whether we can/should update this repo location directly.

We have several other schema fixes as well that we're tracking provisionally in the SARIF SDK repo.

Reply:
> We'll need to ask @dmk42 whether we can/should update this repo location directly.

Since this is not the official location on oasis-open.org, we can update it however we like.  If we want to correct it while we are preparing the errata, to show what the errata will look like, that should be fine.

Reply:
But, we should provide meaningful tags mapping our schema file to the stages on docs.oasis-open.org. This way friendly users do not get annoyed by losing track where our source versions relate to officially (meaning we need a tag for the OS version of the schema. Do we have that already? Unfortunately no. Some member (I was on a break when SARIF became OASIS Standard) of this TC should be able to pin schema versions to the official stages with matching short codes like `os-v2.1.0` which would then provide the identical schema file as <https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/schemas/sarif-schema-2.1.0.json> does.